code/modules/food_and_drinks/kitchen_machinery/gibber.dm

```diff
@@ -331,15 +343,22 @@
 			lturf = T
 
 	for(var/mob/living/carbon/human/H in lturf)
-		if(istype(H) && H.loc == lturf)
+		if(istype(H) && H.loc == lturf && !(locate(/obj/machinery/gibber/autogibber) in H.interaction_queue))
+			target_found = 1
+			H.interaction_queue += src
```

Comment:
Instead, you can simply have autogibbers detect multiple targets at once, and grab the first one that hasn't gotten away.